src/panels/lovelace/cards/energy/hui-energy-devices-detail-graph-card.ts

```diff
@@ -240,6 +250,33 @@ export class HuiEnergyDevicesDetailGraphCard
 
     datasetExtras.push(...processedDataExtras);
 
+    const { summedData, compareSummedData } = getSummedData(energyData);
+
+    const hasProduction =
```

Comment:
```suggestion
    const hasConsmption =
```

Reply:
perhaps `hasEnergySource`?  I don't know that I would call solar production as consumption, seems backwards.

Reply:
True, but we are trying to figure out if we should compute consumption data, thats why I suggested it.

Reply:
I'll go with `showUntracked`, as that's what this boolean is really controlling. `consumption` is a little too overloaded term here I think,